Userland/Libraries/LibGfx/VectorN.h

```diff
@@ -193,15 +193,19 @@ requires(N >= 2 && N <= 4) class VectorN final {
 
     [[nodiscard]] constexpr T length() const
     {
-        if constexpr (N == 2)
-            return AK::hypot(m_data[0] * m_data[0] + m_data[1] * m_data[1]);
-        else if constexpr (N == 3)
-            return AK::sqrt(m_data[0] * m_data[0] + m_data[1] * m_data[1] + m_data[2] * m_data[2]);
-        else
-            return AK::sqrt(m_data[0] * m_data[0] + m_data[1] * m_data[1] + m_data[2] * m_data[2] + m_data[3] * m_data[3]);
+        T squared_sum { 0 };
```

Comment:
This is fine, but not as generic as it *could* be. Since you don't know if `T` can be created from an `int`, you could just default init using `T squared_sum{};` without the `0`. Probably will never cause a problem though since T is meant to be a POD-type.

Reply:
Good point, fixed!